sample_files/pod-security-admission-config.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
add a note here that this came from the official rke2 documentation with a link. also mention that reducing this to minimum required exemptions would be ideal.